src/QuadForm/Herm/LocallyIsometricSublattice.jl

```diff
@@ -147,47 +151,44 @@ function _locally_isometric_sublattice_odd_ramified(M, L, p, P, absolute_map)
   if length(B0) == 0
     LL = lattice(ambient_space(M), _module_scale_ideal(P, pseudo_matrix(M)))
   else
-    _new_pmat = _sum_modules(pseudo_matrix(reduce(vcat, B0)), _module_scale_ideal(P, pseudo_matrix(M)))
+    Ppm = _module_scale_ideal(P, pseudo_matrix(M))
+    _new_pmat = _sum_modules(pseudo_matrix(reduce(vcat, B0)), Ppm)
     _new_pmat = _intersect_modules(_new_pmat, pseudo_matrix(M))
     LL = lattice(ambient_space(M), _new_pmat)
   end
-  @assert genus(LL, p) == c
 
   k, h = ResidueField(order(p), p)
   hext = extend(h, K)
   for j in length(C)-1:-1:1
     c = C[j]
     if all(!(scale(c, i) in [0, 1]) for i in 1:length(c))
-      @assert scale(C[1], 1) - valuation(scale(LL), P) >= 0
       s = div(scale(C[1], 1) - valuation(scale(LL), P), 2)
-      LL = lattice(ambient_space(LL), P^s * pseudo_matrix(LL))
+      LL = lattice(ambient_space(LL), _module_scale_ideal(P^s, pseudo_matrix(LL)))
```

Comment:
Why?

Reply:
This works only for fractional ideals, so for now it was easier than turning `P^s` into a fractional ideal, or create a new method for integral ideals

Reply:
Ah, right, thanks. I remember putting `fractional_ideal()` everywhere :(

Reply:
Yes, that would be a good idea.